source/DSCResources/Resources/SqlServer.ScriptQuery.ps1

```diff
@@ -9,12 +9,14 @@ foreach ($instance in $ServerInstance)
     {
         foreach ($db in $Database)
         {
+            $getScript = '{0} --{1}' -f $rule.GetScript, $db
+
             foreach ($rule in $rules)
             {
-                SqlScriptQuery "$(Get-ResourceTitle -Rule $rule)$instance"
+                SqlScriptQuery "$(Get-ResourceTitle -Rule $rule)$instance_$db"
```

Comment:
I prefer the format operator but will leave it up to you and Brian.

Reply:
So something like?
```PowerShell
$resourceTitle = '{0}{1}_{2}' -f (Get-ResourceTitle -Rule $rule), $instance, $db
SqlScriptQuery "$resourceTitle"
```